src/Http/Place/ReadPlaceRestController.php

```diff
@@ -78,17 +71,16 @@ public function getCalendarSummary($cdbid, Request $request)
         $place = $this->serializer->deserialize($data->getRawBody(), Place::class);
 
         if ($style !== 'html' && $style !== 'text') {
```

Comment:
pretty sure there's a value object hiding here somewhere

Reply:
True but I didn't want to focus on the calendar summary methods so the scope wouldn't get too wide. It's already a lot of changes at once 😄